MAINTAINERS.md

```diff
@@ -4,10 +4,8 @@
 
 | Maintainer | GitHub ID | Affiliation |
 |------------|-----------|-------------|
-| Josh Rosso | [joshrosso](https://github.com/joshrosso) | [VMware](https://www.github.com/vmware/) |
-| Nicholas Seemiller | [seemiller](https://github.com/seemiller) | [VMware](https://www.github.com/vmware/) |
-| Sean McGinnis | [stmcginnis](https://github.com/stmcginnis) | [VMware](https://www.github.com/vmware/) |
-| David vonThenen | [dvonthenen](https://github.com/dvonthenen) | [VMware](https://www.github.com/vmware/) |
-| John McBride | [jpmcb](https://github.com/jpmcb) | [VMware](https://www.github.com/vmware/) |
-| Nolan Brubaker | [nrb](https://github.com/nrb) | [VMware](https://www.github.com/vmware/) |
+| Vuil Lam | [vuil](https://github.com/vuil) | [VMware](https://www.github.com/vmware/) |
```

Comment:
nit:
VMware by Broadcom?

Reply:
Good point.  will do.  Do you think the link https://www.github.com/vmware/ is ok to use?

Reply:
Forgot to push the changes to address this comment.  Did it in #61